### DISCUSSION TITLE
Template Search

### DISCUSSION THREAD

**Discussion by MichaelErmer** [outline/outline#5737] [Ideas]
Currently, there is only a way to search in Templates if there are at most 15 in the system. 

Please add a search bar to the dropdown if more than is displayed.

<img width="290" alt="image" src="https://github.com/outline/outline/assets/628787/aa4a16ee-3d34-4a68-8e69-ce4902840f1c">

**Comment by tommoor** [comment]
> there is only a way to search in Templates if there are at most 15 in the system

Which way are you using to search when there are less than 15 sorry?
